src/Asset/Estate/Estate.model.js

```diff
@@ -63,4 +64,15 @@ export class Estate extends Model {
         WHERE ${BlockchainEventQueries.byArgs('_estateId', estateId)}
           OR (${BlockchainEventQueries.byArgs('_tokenId', estateId)} AND address = ${address})`)
   }
+
+  static async updateEstateDistrictIds() {
```

Comment:
Maybe just `updateDistrictIds` (we're already in Estate.model) or `refreshDistrictIds` because update it's already a verb used in SQL